cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/cost/AwsPricingCache.java

```diff
@@ -83,44 +65,39 @@ public double getPriceForInstanceType(String region, String instanceType) {
                 .getPricePerUnit().getUsd();
     }
 
+    @Cacheable(cacheNames = "awsCostCache", key = "{ #region, #instanceType }")
     public int getCpuCountForInstanceType(String region, String instanceType, ExtendedCloudCredential extendedCloudCredential) {
         PriceListElement priceListElement = getPriceList(region, instanceType);
         return priceListElement.getProduct().getAttributes().getVcpu();
     }
 
+    @Cacheable(cacheNames = "awsCostCache", key = "{ #region, #instanceType }")
     public int getMemoryForInstanceType(String region, String instanceType, ExtendedCloudCredential extendedCloudCredential) {
         PriceListElement priceListElement = getPriceList(region, instanceType);
         String memory = priceListElement.getProduct().getAttributes().getMemory();
         return Integer.parseInt(memory.replaceAll("\\D", ""));
     }
 
+    @Cacheable(cacheNames = "azureCostCache", key = "{ #region, #storageType, #volumeSize }")
```

Comment:
This should be `"awsCostCache"`, I assume.

Reply:
yes, thx!